scripts/template/src/patternfly/{folder}/{name}/{nameDasherized}.scss

```diff
@@ -0,0 +1,26 @@
+// Component variables follow this formula:
+// --{{nameBEM}}__element--modifier--state--PropertyCamelCase
+
+// The value of component scoped variables is always defined by a global variable:
+.{nameBEM} {
+  // Example block variable
+  --{nameBEM}--Color: var(--pf-global--Color--dark);
+
+  // Example element variable
+  --{nameBEM}__element--FontSize: var(--pf-global--FontSize--2xl);
+
+  // Example modifier variable
+  --{nameBEM}--m-danger--BackgroundColor: var(--pf-global--danger-color);
```

Comment:
this looks super. except can we move the `//example modifier variable` comment/css var agovec the element variable block? So the order would be "example block var", "example modifier var", "example element var" - follows the order of the vars used in the SCSS below.

Reply:
Not entirely sure what the "order of the vars used in the SCSS below" is, but I put them in the order block, modifier, element.

Reply:
Oh sorry I mean like this, so that we define the vars in the same order that we use them "color -> background-color -> font-size"

```
// Component variables follow this formula:
// --{{nameBEM}}__element--modifier--state--PropertyCamelCase

// The value of component scoped variables is always defined by a global variable:
.{nameBEM} {
  // Example block variable
  --{nameBEM}--Color: var(--pf-global--Color--dark);

  // Example modifier variable
  --{nameBEM}--m-danger--BackgroundColor: var(--pf-global--danger-color);

  // Example element variable
  --{nameBEM}__element--FontSize: var(--pf-global--FontSize--2xl);

  color: var(--{nameBEM}--Color);

  // Modifier
  &.pf-m-danger {
    background-color: var(--{nameBEM}--m-danger--BackgroundColor);
  }
}

// Element
.{nameBEM}__element {
  font-size: var(--{nameBEM}__element--FontSize);
}
```

Reply:
Cool! That's exactly what I did in my last commit.